Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM quay.io/assemblyline/alpine:3.6 as dev
+
+ENV APP_NAME=lead_pli_gem \
```

Comment:
`APP_NAME` should use dashes instead of underscores. Iif memory serves me right, it will fail to build if that's not case.